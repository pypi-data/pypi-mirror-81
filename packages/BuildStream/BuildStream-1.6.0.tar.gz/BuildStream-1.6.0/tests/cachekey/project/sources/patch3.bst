kind: import
sources:
- kind: patch
  path: files/patches/patch.diff
  directory: usr/bin
  strip-level: 3
