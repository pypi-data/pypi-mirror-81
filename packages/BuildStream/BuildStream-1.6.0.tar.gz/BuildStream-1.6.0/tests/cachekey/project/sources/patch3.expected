f40eb8bab43e2ff8deeb6b5d360d911506be914ef302d8a1de5a431f3d32c222